spark/v3.4/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestCallStatementParser.java

```diff
@@ -174,6 +175,28 @@ public void testCallStripsComments() throws ParseException {
     }
   }
 
+  @Test
+  public void testCallWithEmptyCollectionArg() throws ParseException {
+    CallStatement call =
+        (CallStatement) parser.parsePlan("CALL cat.system.func('test', Map(), Array())");
+    Assert.assertEquals(
```

Comment:
Junit 5 is suggested. You can refer here: https://iceberg.apache.org/contribute/#testing

Reply:
thanks, fixed